test/Microsoft.AspNet.Authentication.Test/Cookies/CookieMiddlewareTests.cs

```diff
@@ -748,12 +749,14 @@ private Task SignOutAsWrong(HttpContext context)
         [Fact]
         public async Task MapWillNotAffectChallenge()
         {
-            var server = TestServer.Create(app =>
+            var builder = new WebApplicationBuilder()
```

Comment:
They were getting unmanageable. You needed another overload for every common builder setting. We wanted to just stick to using the builder directly.